vendor/github.com/portworx/sched-ops/k8s/k8s.go

```diff
@@ -930,6 +934,28 @@ func (k *k8sOps) DescribeDeployment(depName, depNamespace string) (*apps_api.Dep
 	return &dep.Status, err
 }
 
+func (k *k8sOps) UpdateDeployment(deployment *apps_api.Deployment, newRepl *int32) error {
+	logrus.Info("Attempt to update the deployment.")
+	dep, err := k.appsClient().Deployments(deployment.Namespace).Get(deployment.Name, meta_v1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	*dep.Spec.Replicas = *newRepl
+	_, err = k.appsClient().Deployments(dep.Namespace).Update(dep)
```

Comment:
return the updated deployment and err as is to the caller instead of just returning the error

Reply:
FIxed.